### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by morrone**
This is a first pass at updating the REAME before the 4.3.4 release. It could probably use more work, but perhaps it will be a good starting point.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,88 +22,101 @@ applications yield the processor and transported using RDMA-like operations, res
...
-to discover sequence of occurrence patterns of log messages and to perform causal analysis.
+  https://github.com/ovis-hpc/ovis/releases
 
-Baler has been moved out of the OVIS tree. Please check it out at <https://github.com/opengridcomputing/baler>
+The tarball is avialble in the "Assets" section of each release. Be sure to
+download the tarball that has a name of the form "ovis-ldms-X.X.X.tar.gz".
 
-# Obtaining OVIS source code
+The links that are named "Source code (zip)" and "Source code (tar.gz)" are
+automatic GitHub links that we are unable to remove. Yhey will be missing the
```
Comment (tom95858) [inline README.md:42]: *They will be missing

File: README.md [hunk 2]
```diff
@@ -23,88 +22,101 @@ applications yield the processor and transported using RDMA-like operations, res
...
+## Build Dependencies
+
+* autoconf (>=2.63)
+* automake
+* libtool
+* make
+* bison
+* flex
+* gettext development package
+* libevent developmetn package
```
Comment (tom95858) [inline README.md:62]: *development

File: README.md [hunk 3]
```diff
@@ -23,88 +22,101 @@ applications yield the processor and transported using RDMA-like operations, res
...
 For cray-related LDMS sampler plug-in dependencies, please see the man page of the
 plug-in in `ldms/man/`.
 
+### RHEL7/CentOS7 dependencies
+
+RHEL7/CentOS7 systems will require a the following packages at a minimum:
 
-# Building OVIS
+* autoconf
+* automake\
```
Comment (tom95858) [inline README.md:79]: trailing backslash?

File: README.md [hunk 4]
```diff
@@ -23,88 +22,101 @@ applications yield the processor and transported using RDMA-like operations, res
...
 * Ubuntu and friends
 * CentOS and friends
 * Cray XE6, Cray XK, Cray XC
 
 # Unsupported features
-LDMS sampler plugins
+
+The following LDMS sampler plugins are considered unsupported. Use are your own risk:
 * perfevent sampler
 * papi sampler
```
Comment (tom95858) [inline README.md:121]: Do you mean the hweventpapi sampler? The papi sampler is supported.

File: README.md (hunk 2 above)
Comment (narategithub) [reply README.md:62]: @morrone actually, we don't need libevent anymore. I think the comment in `configure.ac` stating that we do was a copy-paste error.

File: README.md (hunk 1 above)
Comment (morrone) [reply README.md:42]: Yep.

File: README.md (hunk 2 above)
Comment (morrone) [reply README.md:62]: OK.

File: README.md (hunk 3 above)
Comment (morrone) [reply README.md:79]: Yes.

File: README.md (hunk 4 above)
Comment (morrone) [reply README.md:121]: I don't know on that one. That was left over from the previous version. I'll assume hweventpapi is the right one and make that change.

**Comment by morrone** [comment]
In addition to the fixes for the review comments, in this version I delete a few of the other README files that are now obsolete. I also delete the obsolete Changelog files in the second commit.

**Pull request merged**